ErpNet.FP.Server/wwwroot/app.js

```diff
@@ -72,6 +77,38 @@ function showAvailablePrinters() {
                 }
                 paymentMappingsContent += '</div>';
 
+                var printerOpionsContent = '';
+                if (printerProps) {
+                    for (var i = 0; i < availablePrinterOptions.length; i++) {
+                        var po = availablePrinterOptions[i];
+                        var optionAvailable = printerProps.printerOptions[po];
+                        if (optionAvailable) {
+                            var printerOptionName = useOfPrinterOptions[i];
+                            var printerOptionHint = hintsForPrinterOptions[i];
+                            var printerOptionDisplayText = displayTextsForPrinterOptions[i];
+                            var inputId = printer.serialNumber + '_po_' + printerOptionName;
+                            var printerOptionValue = '';
+
+                            var pv = printerProps.printerOptions[printerOptionName]
+                            //if (pv) {
+                                if (pv == 'True')
+                                    printerOptionValue = 'checked';
+                            //}
+
+                            printerOpionsContent +=
+                                '<label title ="' + printerOptionHint + '" for="' + inputId + '" style="margin-right: 0;">' +
+                                    printerOptionDisplayText + '</label>' +
+                                '<input type = "checkbox"' + ' id="' + inputId + '" title="' + printerOptionHint + '" ' +
+                                printerOptionValue + ' onFocus="this.select()"/>&nbsp;';
+                        }
+                    }
+                    if (printerOpionsContent.length > 0) {
+                        printerOpionsContent =
+                            '<div class="section dark"><h5 style="padding: 0 em; margin: 0;">Printer options</h5></div>' +
+                            '<div class="section input-group horizontal">' + printerOpionsContent + '</div>';
+                    }
+                }
+
                 var section =
                     '<input type="radio" id="available-section-' + printerId + '" aria-hidden="true" name="available">' +
                     '<label style="overflow:hidden;display:inline-block;text-overflow: ellipsis;white-space: nowrap;" for="available-section-' + printerId + '" aria-hidden="true"><strong>' + printerId + '</strong></label>' +
```

Comment:
I don't know how you arrived at that; aren't standard [bootstrap form controls](https://getbootstrap.com/docs/4.6/components/forms/) going to do the trick?

Reply:
It's Copy/Paste with relevant corrections  from existing code.

Reply:
Possibly it could be done better, but i have basic knowledge about html/css.